frontend/src/pages/CalendarPage/index.js

```diff
@@ -201,11 +207,12 @@ class CalendarPage extends React.Component {
             </Grid>
 
             {selectedEvent && (
```

Comment:
Same here, selectedevent === null

Reply:
Would 

`!(selectedEvent === null)`

 be a good way to write it?

Reply:
selectedEvent !== null works as well